sessionctx/stmtctx/stmtctx.go

```diff
@@ -63,6 +63,7 @@ type StatementContext struct {
 
 	// IsDDLJobInQueue is used to mark whether the DDL job is put into the queue.
 	// If IsDDLJobInQueue is true, it means the DDL job is in the queue of storage, and it can be handled by the DDL worker.
+	InReorgAttribute          bool
```

Comment:
Please add comments for this variable, and please put it after `IsDDLJobInQueue`, line 65 is for `IsDDLJobInQueue`.